### PR TITLE
Sprint 196 & 197 stats and start Sprint 198

### PR DESCRIPTION
**Pull request by Fryguy**
@jrafanie Please review.

### CODE REVIEW

**Review by jrafanie — APPROVED** [review]

File: site/_sprints/197.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Sprint 197 Details
+sprint_number: 197
+slides: https://www.slideshare.net/ManageIQ/sprint-196
```
Comment (chessbyte) [inline site/_sprints/197.md:4]: @Fryguy this is pointing to the wrong sprint

Comment (Fryguy) [reply site/_sprints/197.md:4]: @chessbyte Want a hacktoberfest PR? 😉

Comment (Fryguy) [reply site/_sprints/197.md:4]: Fixed in https://github.com/ManageIQ/manageiq.org/pull/1121

Comment (chessbyte) [reply site/_sprints/197.md:4]: forgot about Hacktoberfest!

**Pull request merged**